.CI/Jenkinsfile

```diff
@@ -32,6 +32,7 @@ pipeline {
           buildingTag()
           anyOf {
             branch 'master'
+            branch 'maint/*'                
```

Comment:
I'm not sure if `'maint/*'` works.  In one of our repos, we have:
```
anyOf {
                buildingTag()
                anyOf {
                  branch 'master'
                  branch 'jenkins' // For testing purposes
                  branch 'maintenance/**'
                }
              }
```
But it's possible they both work and ** is just to also find / in there...

Reply:
I tried searching for documentation, and it doesn't seem easy to find. I added ** just in case.

Reply:
Oh, I finally found the documentation and it seems that just one star would have worked:
https://www.jenkins.io/doc/book/pipeline/syntax/index.html#built-in-conditions
(I also found a lot of GUI-descriptions with regular expression or wildcard; and this explains how it is implemented.)